Tests/Guesser/FilterTypeGuesserTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Sonata\DoctrineORMAdminBundle\Tests\Guesser;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineORMAdminBundle\Guesser\FilterTypeGuesser;
+
+class FilterTypeGuesserTest extends TestCase
+{
+    private $guesser;
+    private $modelManager;
+
+    protected function setUp()
+    {
+        $this->guesser = new FilterTypeGuesser();
+        $this->modelManager = $this->prophesize('Sonata\DoctrineORMAdminBundle\Model\ModelManager');
+        $this->metadata = $this->prophesize('Doctrine\ORM\Mapping\ClassMetadata');
+    }
+
+    /**
+     * @expectException Sonata\DoctrineORMAdminBundle\Model\MissingPropertyMetadataException
```

Comment:
Apparently we recommend `setExpectedException` https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#the-tests

Should be easier to migrate from that than from the annotations.